semantic_release/version/algorithm.py

```diff
@@ -27,7 +27,7 @@
 
 
 def tags_and_versions(
-    tags: Iterable[Tag], translator: VersionTranslator
+    tags: Iterable[Tag], translator: VersionTranslator, only_last_release: bool = False
```

Comment:
I don't think it's necessary to add this, it's not the purpose of `tags_and_versions` at all. There's only one location where you intend to set this to `True`, I don't see why you can't just add the `[0]` there?